macOS 13 + ROOT 6.26.08 broken building from source on homebrew

When building ROOT for Homebrew on macOS 13, the build stops on the XROOTD build, which itself breaks with:

```
[578/589] Linking CXX shared module src/XrdClHttp/libXrdClHttp-5.so
FAILED: src/XrdClHttp/libXrdClHttp-5.so 
: && /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ -std=c++17 -Wc++11-narrowing -Wsign-compare -Wsometimes-uninitialized -Wconditional-uninitialized -Wheader-guard -Warray-bounds -Wcomment -Wtautological-compare -Wstrncat-size -Wloop-analysis -Wbool-conversion -m64 -pipe -W -Wall -Woverloaded-virtual -fsigned-char -fno-common -Qunused-arguments -pthread -stdlib=libc++ -Wno-deprecated-declarations -O3 -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.sdk -bundle -Wl,-headerpad_max_install_names -Wl,-rpath,/usr/local/Cellar/root/6.26.08/lib/root -o src/XrdClHttp/libXrdClHttp-5.so src/XrdClHttp/CMakeFiles/XrdClHttp-5.dir/XrdClHttpPlugInFactory.cc.o src/XrdClHttp/CMakeFiles/XrdClHttp-5.dir/XrdClHttpPlugInUtil.cc.o src/XrdClHttp/CMakeFiles/XrdClHttp-5.dir/XrdClHttpFilePlugIn.cc.o src/XrdClHttp/CMakeFiles/XrdClHttp-5.dir/XrdClHttpFileSystemPlugIn.cc.o src/XrdClHttp/CMakeFiles/XrdClHttp-5.dir/XrdClHttpPosix.cc.o  -Wl,-rpath,/tmp/root-20221102-55129-1mj03so/root-6.26.08/builddir/builtins/xrootd/XROOTD-prefix/src/XROOTD-build/src/XrdCl  /usr/local/lib/libdavix.dylib  src/XrdCl/libXrdCl.3.0.0.dylib && :
Undefined symbols for architecture x86_64:
  "XrdOucCRC::Calc32C(void const*, unsigned long, unsigned int)", referenced from:
      XrdCl::PgReadSubstitutionHandler::HandleResponse(XrdCl::XRootDStatus*, XrdCl::AnyObject*) in XrdClHttpFilePlugIn.cc.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

This is after both a macOS bump (13) and a ROOT bump (6.26.08). Building this locally: https://github.com/Homebrew/homebrew-core/pull/114543 with

```bash
brew install root --build-from-source -v --keep-tmp
```

The first error I see there is an error building the Python bindings. I will try to reproduce the other error as well on macOS 12.
```
:info:build Version:                5.5.1
:info:build Traceback (most recent call last):
:info:build   File "/opt/local/var/macports/build/_Users_chris_Projects_MacPorts_ports_science_xrootd/xrootd/work/build/b
indings/python/setup.py", line 111, in <module>
:info:build     setup( name             = 'xrootd',
:info:build   File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/setuptools/
_distutils/core.py", line 185, in setup
:info:build     return run_commands(dist)
:info:build   File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
:info:build     dist.run_commands()
:info:build   File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
:info:build     self.run_command(cmd)
:info:build   File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
:info:build     cmd_obj = self.get_command_obj(command)
:info:build   File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 861, in get_command_obj
:info:build     cmd_obj = self.command_obj[command] = klass(self)
:info:build   File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/setuptools/__init__.py", line 158, in __init__
:info:build     super().__init__(dist)
:info:build   File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 61, in __init__
:info:build     raise TypeError("dist must be a Distribution instance")
:info:build TypeError: dist must be a Distribution instance
:info:build make[2]: *** [bindings/python/python_bindings] Error 1
:info:build make[2]: Leaving directory `/opt/local/var/macports/build/_Users_chris_Projects_MacPorts_ports_science_xrootd/xrootd/work/build'
:info:build make[1]: *** [bindings/python/CMakeFiles/python_target.dir/all] Error 2
:info:build make[1]: *** Waiting for unfinished jobs....
```

--- a/src/XrdClHttp/CMakeLists.txt
+++ b/src/XrdClHttp/CMakeLists.txt
@@ -11,6 +11,6 @@ set(PLUGIN_NAME "XrdClHttp-${PLUGIN_VERSION}")
 
 add_library(${PLUGIN_NAME} MODULE ${libXrdClHttp_sources})
 
-target_link_libraries(${PLUGIN_NAME} ${Davix_LIBRARIES} XrdCl)
+target_link_libraries(${PLUGIN_NAME} ${Davix_LIBRARIES} XrdCl XrdUtils)
 
 install(TARGETS ${PLUGIN_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})

